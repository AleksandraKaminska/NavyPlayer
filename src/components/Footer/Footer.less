.Footer {
  border-top: 1px solid #33363b;
  p {
    margin: 0
  }
  .deezerLogo {
    height: 1em;
    width: auto;
  }
}
