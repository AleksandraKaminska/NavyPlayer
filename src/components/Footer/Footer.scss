.Footer {
  p {
    margin: 0
  }
  img {
    height: 1em;
    width: auto;
  }
}
