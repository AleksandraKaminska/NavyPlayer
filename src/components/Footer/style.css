footer{width:100%;height:3vh;display:flex;flex-wrap:wrap;justify-content:flex-start;align-items:center;align-items:flex-end;font-size:.7em;padding:0 1vw}footer img{height:.7em;width:auto}@media (min-width: 870px){footer{display:block;position:static;height:4vh;padding:.5em;cursor:default;font-size:1em;background:transparent;z-index:0}footer p{visibility:visible}footer img{height:1em;position:static}}
