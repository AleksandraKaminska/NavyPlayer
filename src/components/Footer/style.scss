@import 'sass/modules/_mixins.scss';

footer {
  font-size: 0.7em;
  position: absolute;
  bottom: 0.3em;
  right: 1em;
  img {
    height: 0.7em;
    width: auto;
    vertical-align: middle;
  }
  @media (min-width: 768px) {
    display: block;
    cursor: default;
    font-size: 0.8em;
    img {
      height: 0.8em;
    }
  }
}
