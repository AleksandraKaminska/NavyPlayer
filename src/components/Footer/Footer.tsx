import React from 'react'
import { Layout } from 'antd'
import Player from '../Player/Player'
import './Footer.less'

const Footer = () => (
  <Layout.Footer className="Footer">
    <Player />
  </Layout.Footer>
)

export default Footer
