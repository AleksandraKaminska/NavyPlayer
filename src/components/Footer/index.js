import React from 'react'
import { login } from 'helperFunctions'
import './style.css'

const Footer = () => (
  <footer>
    <p onClick={login}>
      Powered by <a href='http://www.deezer.com/pl/' target='_blank' rel='noopener noreferrer'>
        <img src='/assets/images/DZ_Logo.png' alt='Deezer Logo' />
      </a>
    </p>
  </footer>
)

export default Footer
