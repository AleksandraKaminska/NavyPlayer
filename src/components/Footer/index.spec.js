import React from 'react'
import Footer from './index'
import renderer from 'react-test-renderer'

describe('components', () => {
  describe('Footer', () => {
    it('should render self', () => {
      const rendered = renderer.create(<Footer />)
      expect(rendered.toJSON()).toMatchSnapshot()
    })
  })
})
