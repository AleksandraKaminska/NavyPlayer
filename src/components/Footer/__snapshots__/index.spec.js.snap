// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Footer should render self 1`] = `
<footer>
  <p
    onClick={[Function]}
  >
    Powered by 
    <a
      href="http://www.deezer.com/pl/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Deezer Logo"
        src="/assets/images/DZ_Logo.png"
      />
    </a>
  </p>
</footer>
`;
